tests/python/unittest/test_loss.py

```diff
@@ -224,8 +224,9 @@ def test_ctc_loss_train():
             optimizer='adam')
     assert mod.score(data_iter, eval_metric=mx.metric.Loss())[0][1] < 10
 
-
-@with_seed(1234)
+# @vrakesh: Get rid of fixed seed, as flakiness cannot be reporduced
+# Issue tracked at https://github.com/apache/incubator-mxnet/issues/11700
```

Comment:
Feel free to remove those 2 lines and simply link the original issue in the PR description

Reply:
done